docs/persistent_compilation_cache.md

```diff
@@ -80,6 +81,23 @@ cc.set_cache_dir("/tmp/jax_cache")
 
 Note that both criteria need to be satisfied for a function to be cached.
 
+### Additional caching
+
+XLA supports additional caching mechanism which can be enabled alongside JAX's
+persistent compilation cache to further improve recompilation time.
+
+* `jax_persistent_cache_enable_xla_caches`: Possible values:
+
+   * `all`: enable all XLA caching features
+
+   * `none`: don't enable any extra XLA caching features
+
+   * `xla_gpu_kernel_cache_file`: only enable the kernel cache
```

Comment:
What is the issue with that one? Should we document it here?
Is it just that it doesn't work, or that it give hash collision, or crashes?

Reply:
There is a crash which looks like this:
```
2024-07-30 18:04:57.120490: I external/xla/xla/stream_executor/cuda/cuda_executor.cc:272] getting function input_concatenate_fusion from module 0x55d95b91f8d0
E0730 18:04:57.120755   52461 pjrt_stream_executor_client.cc:3067] Execution of replica 0 failed: NOT_FOUND: Failed to get module function: CUDA_ERROR_NOT_FOUND: named symbol not found
```

@sergachev is taking a look at it and found it can be reproduced using `bazel test --test_env=XLA_FLAGS="--xla_gpu_enable_llvm_module_compilation_parallelism --xla_gpu_kernel_cache_file=/dev/shm/xla.kernel.cache" tests/compilation_cache_test_gpu`

Reply:
Note, that issue is fixed in XLA: https://github.com/openxla/xla/pull/15998
But I think it is better to not able all XLA caches at the same time for better testing.
We can have a follow up PR to expand it after the next JAX releases.